internal/pkg/skuba/deployments/ssh/kubernetes.go

```diff
@@ -105,3 +109,20 @@ func kubernetesRestartServices(t *Target, data interface{}) error {
 	_, _, err := t.ssh("systemctl", "restart", "crio", "kubelet")
 	return err
 }
+
+func kubernetesWaitForKubelet(t *Target, data interface{}) error {
+	clientSet, err := kubernetes.GetAdminClientSet()
+	if err != nil {
+		return errors.Wrap(err, "Error getting client set")
+	}
+	for i := 0; i < 300; i++ {
+		_, err := clientSet.CoreV1().Nodes().Get(t.target.Nodename, metav1.GetOptions{})
+		if err != nil {
+			klog.V(1).Infof("failed to get node %s, continuing...", t.target.Nodename)
+		} else {
+			return nil
+		}
+		time.Sleep(1 * time.Second)
+	}
```

Comment:
What happens if after all this time we are still getting errors? (let's consider the case in which the Kubelet is utterly broken).  We should be able to return a proper error (i.e. a timeout error) instead of just `nil`

Reply:
done